docs/architecture/grpc-for-wcf-developers/call-credentials.md

```diff
@@ -69,16 +69,10 @@ Next, add the Authorization service, which controls access to the system:
 > [!TIP]
 > Authentication and authorization are two separate steps. You use authentication to determine the user's identity. You use authorization to decide whether that user is allowed to access various parts of the system.
 
-Now add the authentication and authorization middleware to the ASP.NET Core pipeline in the `Configure` method:
+Now add the authentication and authorization middleware to the ASP.NET Core pipeline in the `Program.cs`:
```

Comment:
```suggestion
Now add the authentication and authorization middleware to the ASP.NET Core pipeline in the _Program.cs_:
```

When referencing a file, the text should be italic.

Reply:
Ok sure